Services/CO.CDP.Forms.WebApi/UseCase/UpdateFormSectionAnswersUseCase.cs

```diff
@@ -23,9 +23,51 @@ public async Task<bool> Execute((Guid formId, Guid sectionId, Guid answerSetId,
         var section = await formRepository.GetSectionAsync(formId, sectionId)
             ?? throw new UnknownSectionException($"Unknown section {sectionId} in form {formId}.");
 
-        var sharedConsent = await formRepository.GetSharedConsentDraftWithAnswersAsync(formId, organisationId)
-                            ?? CreateSharedConsent(organisation, section.Form);
+        var sharedConsent = await formRepository.GetSharedConsentWithAnswersAsync(formId, organisationId);
+        if (sharedConsent == null)
```

Comment:
Should it include sharedConsent.SubmissionState == Persistence.SubmissionState.Draft ?

Reply:
yes, could either not exist or be in Draft or Submitted state

Reply:
> Should it include sharedConsent.SubmissionState == Persistence.SubmissionState.Draft ?

It should fetch the one created last. If it's not a draft, it will be duplicated before it's updated.